framework/src/restart/DataIO.C

```diff
@@ -326,7 +326,11 @@ dataStore(std::ostream & stream,
           std::unique_ptr<libMesh::NumericVector<Number>> & v,
           void * context)
 {
-  mooseAssert(v, "Null vector");
+  bool have_vector = v.get();
```

Comment:
Can you add a comment here as to why you needed this? As it's a bit inconsistent with other nullptr things